### PR TITLE
New Industry Theme: Foreign Exchange Market Alerts

### PR DESCRIPTION
**Pull request by pdruley**
This is a new use case based on using currency pair data that one would see in foreign exchange markets. It's specifically designed for Confluent Cloud and includes some basic instructions for using managed ksqlDB. API keys and secrets have been cleansed. 

I messed up the initial commit because I hadn't switch to a new branch and so I reverted and then generated a new commit. Of course I also forgot how to unstage files and accidentally removed my files, so I had to checkout my mistaken commit to get the previous files. It was kind of a mess but I think it's all good now.

### CODE REVIEW

File: industry-themes/fx_client_services/ksqlDB-cloud-setup.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#create kafka cluster in UI
+
+#creata a kafka API key/secret in UI
+
+#get bootstrap server
+pkc-ep9mm.us-east-2.aws.confluent.cloud:9092
```
Comment (rmoff) [inline industry-themes/fx_client_services/ksqlDB-cloud-setup.sh:6]: should this be masked? It's not of use on its own, but it is confusing

File: industry-themes/fx_client_services/ksqlDB-cloud-setup.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#create ksqlDB cluster in UI
+
+#login into ccloud cli
+ccloud login
+
+#check default environment 
+ccloud environment list
+
+#set environment
+ccloud environment use env-9wmxm
```
Comment (rmoff) [inline industry-themes/fx_client_services/ksqlDB-cloud-setup.sh:24]: should this be masked? It's not of use on its own, but it is confusing

File: industry-themes/fx_client_services/ksqlDB-cloud-setup.sh [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+#get kafka cluster list
+ccloud kafka cluster list
+
+#set default cluster
+ccloud kafka cluster use lkc-mz7g2
```
Comment (rmoff) [inline industry-themes/fx_client_services/ksqlDB-cloud-setup.sh:30]: should this be masked? It's not of use on its own, but it is confusing

File: industry-themes/fx_client_services/ksqlDB-cloud-setup.sh (hunk 1 above)
Comment (pdruley) [reply industry-themes/fx_client_services/ksqlDB-cloud-setup.sh]: True, just added a commit with more masking.

**Pull request merged**